docker-compose.yml

```diff
@@ -16,11 +16,10 @@ services:
       - ./prometheus/:/etc/prometheus/
       - prometheus_data:/prometheus
     command:
-      - '-config.file=/etc/prometheus/prometheus.yml'
-      - '-storage.local.path=/prometheus'
-      - '-alertmanager.url=http://alertmanager:9093'
```

Comment:
why got alertmanager config removed ?

Reply:
Good catch. The option is not supported anymore as of 2.0. I'm pushing a commit with the new way of configuring alert manager.

Reply:
See https://prometheus.io/docs/prometheus/2.0/migration/
in particular https://prometheus.io/docs/prometheus/2.0/migration/#alertmanager-service-discovery

Reply:
Back to life
<img width="1207" alt="screen shot 2017-11-10 at 14 37 46" src="https://user-images.githubusercontent.com/2582479/32660859-d79cd2b2-c624-11e7-9ba1-726ca7545fb5.png">